python/feathub/processors/flink/table_builder/aggregation_utils.py

```diff
@@ -41,12 +41,16 @@ def __init__(
         expr: str,
         agg_func: AggFunc,
         window_size: timedelta,
+        limit: int,
```

Comment:
Given that limit comes from transform.limit and its value can be none, should we make it `Optional[int]`?